src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -173,6 +173,22 @@ scans:
 
 #### Suffix
 
+Based on their function, suffixes are divided into three classes:
```

Comment:
Reorder list to 1, 3, 2 so that the groupings go last - they're the most complex concept.